src/test/java/commonmark_java_renderer/CommonmarkRendererTest.java

```diff
@@ -0,0 +1,15 @@
+package commonmark_java_renderer;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class CommonmarkRendererTest {
+
+	@Test
+	public void test_italics() {
+		CommonmarkRenderer cr = new CommonmarkRenderer();
+		assertEquals(cr.render("*test*"), "<p><em>test</em></p>\n");
```

Comment:
Please change the "<p><em>test</em></p>\n" part.
Hit the GFM API with "*test*". Take what comes back. Create an italics.html file in src/test/resources
Assert that the file contents are the same as cr.render("*test*").
You can create a Helper class (like FileReaderHelper) to read the html file contents in.

Reply:
Sorry, just saw this comment. I will add this in.